swap/src/cli/api/request.rs

```diff
@@ -1216,133 +1256,268 @@ pub async fn resolve_approval_request(
     Ok(ResolveApprovalResponse { success: true })
 }
 
-fn qr_code(value: &impl ToString) -> Result<String> {
-    let code = QrCode::new(value.to_string())?;
-    let qr_code = code
-        .render::<unicode::Dense1x2>()
-        .dark_color(unicode::Dense1x2::Light)
-        .light_color(unicode::Dense1x2::Dark)
-        .build();
-    Ok(qr_code)
+pub async fn fetch_quotes_task(
+    rendezvous_points: Vec<Multiaddr>,
+    namespace: XmrBtcNamespace,
+    sellers: Vec<Multiaddr>,
+    identity: identity::Keypair,
+    db: Option<Arc<dyn Database + Send + Sync>>,
+    tor_client: Option<Arc<TorClient<TokioRustlsRuntime>>>,
+    tauri_handle: Option<TauriHandle>,
+) -> Result<(
+    tokio::task::JoinHandle<()>,
+    ::tokio::sync::watch::Receiver<Vec<SellerStatus>>,
+)> {
+    let (tx, rx) = ::tokio::sync::watch::channel(Vec::new());
+
+    let rendezvous_nodes: Vec<_> = rendezvous_points
+        .iter()
+        .filter_map(|addr| addr.split_peer_id())
+        .collect();
+
+    let fetch_fn = list_sellers_init(
```

Comment:
Use the sellers variable from above here. This'll require some refactoring in list_sellers.rs to also accept the list of makers (in addition to rendezvous points and the peers from the database

We need this to be able to swap with peers we only know of from the public registry.

@Einliterflasche can help here